cloud/readme.md

```diff
@@ -0,0 +1,63 @@
+# UDMI terraform configuration files
```

Comment:
should this go in docs/cloud/gcp?

Reply:
> 



> How GCP-specific is all this? If it's "mostly" GCP at this point, I'd just go ahead and put it in cloud/gcp to keep the structure with (e.g.) docs/cloud/gcp -- not a big deal, we can always move it later if desired.

Yup, good idea about moving all to cloud/gcp, I've done it

Reply:
> should this go in docs/cloud/gcp?

I'm a bit ambivalent about this. I know we have all the other documentation inside docs, so I guess the answer is yes, but it's also nice to get into that folder and read how to use it. What do you think?

Reply:
I checked with @noursaidi and we've added a link [here](https://github.com/faucetsdn/udmi/blob/b3f9d8c545253d23f2c1cde6f569b8dc4489fef9/docs/cloud/gcp/readme.md?plain=1#L8)

Reply:
There is also the [`dashboard`](https://github.com/faucetsdn/udmi/tree/master/dashboard) which is also GCP only, so a `cloud/gcp` to evetnually house the `dashboard` directory makes sense

Reply:
I'm not sold on the location of the readme.md file. Part of the reason to have all the docs in one place is for discoverability and automatic tool processing. As soon as they're spread all over then things get a lot more complicated!  It's generally a Bad Idea (TM) to have links that go up-and-over except in some categorical cases (i.e. from documentation to examples, or documentation to code).

Reply:
I've just moved the docs to `docs/cloud/gcp` where @grafnu likes them to be. My rationale is that it's easier to find a helpful note (especially for use and installation) directly in the folder of the code, rather than have to search for it in the docs folder. More formal documentation can still be in docs, but it has a different purpose. Anyway, moved to `docs/cloud/gcp` now.